base/io.jl

```diff
@@ -167,24 +167,50 @@ The text is assumed to be encoded in UTF-8.
 readuntil(filename::AbstractString, args...) = open(io->readuntil(io, args...), filename)
 
 """
-    readline(stream::IO=STDIN)
-    readline(filename::AbstractString)
+    readline(stream::IO=STDIN, chomp::Bool=false)
+    readline(filename::AbstractString, chomp::Bool=false)
```

Comment:
The plan was to merge this first, and then open a PR to deprecate the one-argument form of  `readline` in 0.6. Then the default would be changed in 1.0.

We could make the breaking change if you say packages are fine, but the deprecation would allow people to notice they no longer need to remove trailing newlines, which will make code simpler/faster. Might be worth the inconvenience, not sure.